sklearn/semi_supervised/tests/test_label_propagation.py

```diff
@@ -43,9 +43,8 @@ def test_distribution():
     for estimator, parameters in ESTIMATORS:
         clf = estimator(**parameters).fit(samples, labels)
         if parameters['kernel'] == 'knn':
-            continue    # unstable test; changes in k-NN ordering break it
```

Comment:
I'm a little worried that changes like this might break on platforms checked in the release process.

Reply:
I'll revert these changes. I'm not confident about them anyway, since I don't really know the algorithm well.